jsk_arc2017_baxter/package.xml

```diff
@@ -5,6 +5,7 @@
   <description>Baxter-specific programs for Amazon Robotics Challenge 2017 at JSK Lab.</description>
 
   <maintainer email="www.kentaro.wada@gmail.com">Kentaro Wada</maintainer>
+  <maintainer email="shingogo@hotmail.co.jp">Shingo Kitagawa</maintainer>
```

Comment:
ok, in that idea, maintainer in `jsk_2016_01_baxter_apc`'s authors are totally wrong.